app/src/main/res/values/strings.xml

```diff
@@ -410,4 +410,7 @@
   <!-- notification channel names [CHAR LIMIT=40] -->
   <string name="notification_channel_download">Quran Downloads</string>
   <string name="notification_channel_audio">Quran Recitation</string>
+
+  <!-- Widgets -->
+  <string name="no_bookmarks">No bookmarks currently set</string>
```

Comment:
Can you please merge this with `bookmarks_list_empty` string? Perhaps, simply by rename that one. If you want, you  may rephrase it also. E.g.:

```
<string name="no_bookmarks">No bookmarks yet</string>
```

Reply:
If it's fine with you, I've deleted `no_bookmarks` and am just using `bookmarks_list_empty` as-is. Wording seems good enough to me, and not changing the wording means no updating translations.

Also, I couldn't find any other code using `bookmarks_list_empty`, so it's good you've spotted this.